.github/workflows/image_build.yaml

```diff
@@ -0,0 +1,31 @@
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+    branches:
+      - "main"
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: kubeconfig
+        run: 'echo -e "$KUBECONFIG" > $GITHUB_WORKSPACE/project-infra/.kubeconfig'
+        shell: bash
+        env:
+          KUBECONFIG: ${{secrets.KUBECONFIG}}
+      - name: build image
+        shell: bash
+        env:
+          QUAY_TOKEN: ${{secrets.QUAY_TOKEN}}
+          KUBECONFIG: ${{secrets.KUBECONFIG}}
+          CONTROLLER_IMG: "quay.io/capk/capk-manager" 
```

Comment:
Looking at the make file, I think you need to set the `REGISTERY=quay.io/capk` and `TAG=dev-latest` env vars instead of the CONTROLLER_IMG env var directly.

Reply:
quay.io/capk is the path of the organization.
quay.io/capk/capk is the path of the repo.
This is the command that runs in "docker-push" in the Makefile:
docker push $(CONTROLLER_IMG)-$(ARCH):$(TAG)

Should I changed it or add another command?